scripts/qa_crowbarsetup.sh

```diff
@@ -3131,7 +3142,7 @@ function onadmin_proposal
     safely oncontroller check_crm_failcounts
     # For all remaining proposals, check for HA failures after each deployment
     for proposal in horizon ceilometer heat manila trove \
-        barbican magnum sahara murano aodh tempest; do
+        designate barbican magnum sahara murano aodh tempest; do
```

Comment:
Don't we propose to deploy the barclamps from the list from top to bottom? then (assuming https://github.com/crowbar/crowbar-openstack/pull/1707 gets merged), designate would come after cinder

Reply:
No. Cinder does **not** depend on Designate. I thought we recommend to deploy the barclamps in the given order (you can skip barlamps, of course). But I can't find that documentation so just ignore my comment.

Reply:
Ah ok :smile:

Reply:
@toabctl I think this list is mostly unsorted. I can deploy it earlier if you like

Reply:
if it doesn't matter we can leave it as it is.